doc/sphinxext/numpy_ext/numpydoc.py

```diff
@@ -19,8 +19,7 @@
 
 from __future__ import unicode_literals
 
-import sys # Only needed to check Python version
```

Comment:
changes in numpy_ext are not really helpful. we need so sync with their master and do changes upstream.